src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
+  []
+  (-> "./csv"
+      clojure.java.io/file
+      (remove #(.isDirectory %1))
+      (map #(.getName %1))
+      (filter #(.endsWith %1 ".csv"))))
+
+(defn- csv-name->fname
```

Comment:
The other occasion I've seen this is in `clj->js` to convert. Will change though